sky/skylet/providers/gcp/constants.py

```diff
@@ -9,6 +9,47 @@
     "mtu": 1460,
     "routingConfig": {"routingMode": "GLOBAL"},
 }
+# Required firewall rules for SkyPilot to work.
+# Note: there are more than one acceptable required rules.
+FIREWALL_RULES_REQUIRED = [
+    [
+        {
+            "direction": "INGRESS",
+            "allowed": [
+                {'IPProtocol': 'tcp', 'ports': ['0-65535']},
+                {'IPProtocol': 'udp', 'ports': ['0-65535']},
+                {'IPProtocol': 'icmp'}
```

Comment:
Are we going to remove this?

Reply:
yeah we can remove it but I thought if we have chance to create a new VPC, allowing `ping` to work doesn't hurt. but I don't have strong opinion here.

Reply:
Ahh sorry I misread the comment earlier.
I think we can't remove this because these three rules are the default VPC generated by GCP.
if we remove this, then skypilot will always create a new `skypilot-vpc`.